pages/community/[community].tsx

```diff
@@ -0,0 +1,62 @@
+import { observer } from 'mobx-react';
+import { InferGetServerSidePropsType } from 'next';
+import { cache, compose, errorLogger, translator } from 'next-ssr-middleware';
+import { FC } from 'react';
+import { Container } from 'react-bootstrap';
+
+import { CommunityMemberList } from '../../components/Community/MemberList';
+import PageHead from '../../components/Layout/PageHead';
+import { i18n } from '../../models/Base/Translation';
+import {
+  CommunityMember,
+  CommunityMemberModel,
+} from '../../models/Community/CommunityMember';
+
+const { t } = i18n;
+
+export const getServerSideProps = compose<
+  { community: string },
+  {
+    list: CommunityMember[];
+    community: string;
+  }
+>(
+  cache(),
+  errorLogger,
+  translator(i18n),
+  async ({ params: { community } = {} }) => {
+    const list = await new CommunityMemberModel().getAll({ community });
+    return { props: JSON.parse(JSON.stringify({ list, community })) };
+  },
+);
+
+const CommunityMemberListPage: FC<
+  InferGetServerSidePropsType<typeof getServerSideProps>
+> = observer(({ list, community }) => (
+  <Container className="py-5">
+    <PageHead title={`${community}${t('community')}`} />
+    {list[0] ? (
+      <section className="my-5 text-center">
+        <h2 className="mb-5 text-center">{t('community_member')}</h2>
+        <CommunityMemberList list={list} />
+      </section>
+    ) : (
+      <section className="text-center">
+        <h3 className="my-5">{t('add_member')}</h3>
+        <a href="https://kaiyuanshe.feishu.cn/share/base/form/shrcnogj5LPzlaiUkFaKpVbxNXe">
+          {t('member_register')}
+        </a>
+        <br />
+        <a href="https://kaiyuanshe.feishu.cn/share/base/form/shrcnAyfE76AHnwtJ8P1fO7avaf">
+          {t('community_register')}
+        </a>
+        <br />
+        <a href="https://kaiyuanshe.feishu.cn/share/base/form/shrcnaAyjtchedloWwloeSQxNZe">
+          {t('activity_register')}
+        </a>
+      </section>
+    )}
```

Comment:
```suggestion
  <section className="my-5 text-center">
    {list[0] ? (
      <>
        <h2 className="mb-5">{t('community_member')}</h2>
        <CommunityMemberList list={list} />
      </>
    ) : (
        <h2>{t('add_member')}</h2>
    )}
        <Button
          size="lg"
          href="https://kaiyuanshe.feishu.cn/share/base/form/shrcnogj5LPzlaiUkFaKpVbxNXe">
          {t('member_register')}
        </Button>
      </section>
```
1. 除了文章内容，布局不要用 `<br />`。
2. 社区成员页只显示成员申请表，社区成立申请表只能放在社区列表页，活动发起表暂时不放出来，只给社区组织者用。